src/gilded_rose.spec.js

```diff
@@ -168,7 +167,7 @@ describe('generating ordered lists of items', () => {
     const updatedItemsList = updateQuality(itemsList);
     let updatedItemsHTML = '<ol>';
     updatedItemsList.forEach((item) => {
-      updatedItemsHTML += `<li>${JSON.stringify(item)}</li>`
+      updatedItemsHTML += `<li>${JSON.stringify(item)}</li>`;
```

Comment:
Just a quick note on curating your commits before a PR: when we talked about this, I mentioned that I don't curate my commits after opening a PR. What I _should_ have said is that I don't curate my commits after a reviewer has started looking at it (because the reason we avoid changing the commits while the PR is in review is that if we do that, we might lose track of requested changes from our reviewer).

Looking through the order of commits here, I was wondering if you pushed your changes up, opened a PR, and then saw linting failures. As long as no one has started reviewing your PR yet, feel free to squash that fix into the commit that created the error. This way, you eliminate one commit that the review doesn't have to go over and reduce noise a bit.